connect/eaas/core/decorators.py

```diff
@@ -438,6 +439,44 @@ def wrapper(func):
     return wrapper
 
 
+def devops_pages(pages: List[Dict]):
+    """
+    Class decorator for Web Application that declare a list of
+    devops pages that will be displayed.
+    These pages are shown as additional tabs in the detail view of a specific extension
```

Comment:
It is not in the detail view of a specific extension, is in the main page of the devops module